frame/composable-traits/src/vamm.rs

```diff
@@ -15,23 +15,26 @@ use sp_std::fmt::Debug;
 pub trait Vamm {
 	/// The balance type for an account.
 	type Balance;
+
 	/// Signed fixed point number implementation
 	type Decimal: FixedPointNumber;
+
 	/// Configuration for creating and initializing a new vAMM instance. May be used in extrinsic
 	/// signatures
 	type VammConfig: FullCodec + MaxEncodedLen + TypeInfo + Debug + Clone + PartialEq;
+
```

Comment:
Is there a guideline for whether or not to separate trait associated types with a blank line? It seems some pallets do it while others don't. I'm okay with either to be honest

Reply:
I'm not aware of such a guideline for it, but I have a personal preference in favor of black lines between types, since otherwise the code and docs seems a bit clunky.
(No prob to remove the blank lines if needed tho)

Reply:
No problems, let's keep it this way. Ready to merge then